packages/governance/README.md

```diff
@@ -9,18 +9,24 @@ connected to so that voters can verify that their votes mean what they say and
 will be tabulated as expected.
 
 Any occasion of governance starts with the creation of an Electorate. Two kinds
-exist currently that represent committees and stakeholders (Stakeholder support
-is in review). The electorate may deal with many questions governing many
-things, so the electorate has to exist before any questions can be posed.
+exist currently that represent committees (elected or appointed) and
+stakeholders (Stakeholder support is in review). The electorate may deal with
+many questions governing many things, so the electorate has to exist before any
+questions can be posed.
 
-The next piece to be created is an ElectionManager. (A Contract Governor, is a
-particular example, discussed below). An ElectionManager is tied to a particular
+The next piece to be created is an ElectionManager. (Contract Governor, one
+implementation, is discussed below). An ElectionManager is tied to a particular
 Electorate. It supports creation of questions, can manage what happens with the
 results, and may limit the kinds of questions it can handle. The ElectionManager
 is also responsible for specifying which VoteCounter will be used with any
 particular question. Different VoteCounters will handle elections with two
-positions or more, with plurality voting, single-transferable-vote, or
-instant-runoff-voting.
+positions or more. Separate vote counters will be available to support
+[majority decisions](https://en.wikipedia.org/wiki/Majority_rule),
+[approval voting](https://en.wikipedia.org/wiki/Approval_voting), and
+[proportional representation](https://en.wikipedia.org/wiki/Proportional_representation),
+as well as [quadratic](https://en.wikipedia.org/wiki/Quadratic_voting),
+[instant runoff](https://en.wikipedia.org/wiki/Category:Instant-runoff_voting),
+and more esoteric approaches.
```

Comment:
I don't see approval voting in our issues list. By "will be available" do we mean "will be provided by Agoric"? Or "could be written"?